applications/GeoMechanicsApplication/tests/test_transient_thermal.py

```diff
@@ -263,6 +263,14 @@ def test_thermal_point_flux_3D2N(self):
     def test_thermal_point_flux_3D3N(self):
         temperature = self.simulate_thermal_case('test_thermal_heat_flux_line_element/test_thermal_point_flux_3D3N')
         self.assertAlmostEqual(self.etalon_value6, temperature[0])
+    
+    def test_thermal_filter_element_2D2N(self):
+        temperature = self.simulate_thermal_case('test_thermal_filter_element/test_thermal_filter_element_2D2N')
+        self.assertAlmostEqual(7.679491925, temperature[22])
```

Comment:
Please introduce a variable with clear name, for example, etalon_temperature_at_node80=7.67....